uuid/v4.ts

```diff
@@ -0,0 +1,27 @@
+import { NIL_UUID } from "./mod.ts";
+
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+/**
+ * Super-small uuid generation.
+ *
+ * Based on
+ *  https://gist.github.com/jed/982883
```

Comment:
not sure if this should stay here since the algorithm is pretty straightforward and the license on that gist does not require it at all

Reply:
killed it